internal/hcsoci/resources.go

```diff
@@ -125,6 +132,22 @@ func ReleaseResources(ctx context.Context, r *Resources, vm *uvm.UtilityVM, all
 			}
 			r.scsiMounts = nil
 		}
+
+		if r.tmpVhdPath != "" {
+			if err := os.Remove(r.tmpVhdPath); err != nil {
+				if os.IsPermission(err) {
+					// May still be mounted. Try to force the dismount.
```

Comment:
Are there legitimate cases where the VHD won't be unmounted? Or are we just trying to be careful in case there is a bug that prevents it?

Reply:
Oh actually this code is only needed for WCOW. I will remove it.